### ISSUE TITLE
how can I train on my own dataset

### ISSUE THREAD

**Issue by tigerdhl** [liuruijin17/LSTR#46]
@liuruijin17  How can I use in my owndata?  I create a json file, and write in my data path, 
![Dingtalk_20210413093733](https://user-images.githubusercontent.com/29905731/114485322-4c7b8180-9c3e-11eb-8cda-23fe13ba0f2e.jpg)

I use the command:
"python test.py LSTR --testiter 500000 --modality eval --split testing --debug"
and get the wrong result as follow:
![AEBSdata_1492626047222176976_0_3](https://user-images.githubusercontent.com/29905731/114485327-50a79f00-9c3e-11eb-852e-9c42446dd1df.jpg)

**Comment by tigerdhl** [comment]
I also use the command
"python test.py LSTR --testiter 500000 --modality images --image_root ./ --debug"
put my data in the directory "./images",and get the same result like this,
![3](https://user-images.githubusercontent.com/29905731/114489315-8308ca80-9c45-11eb-90fb-e8e5146ca42e.jpg)

**Comment by liuruijin17** [comment]
Hi, @tigerdhl 
Two solutions:

1. In the first image, the dotted lines show that your **ground truth** labels do not fit lanes well, which might be caused by the draw_annotation function. You may debug the draw_annotation function yourself to correct the visualization of **ground truth** labels.

3. In both images, the tested results by TuSimple pre-trained LSTR also do not fit lanes well. The reason why this happens has been discussed in #8 #27.

**Comment by tigerdhl** [comment]
> Hi, @tigerdhl
> Two solutions:
> 
> 1. In the first image, the dotted lines show that your **ground truth** labels do not fit lanes well, which might be caused by the draw_annotation function. You may debug the draw_annotation function yourself to correct the visualization of **ground truth** labels.
> 2. In both images, the tested results by TuSimple pre-trained LSTR also do not fit lanes well. The reason why this happens has been discussed in #8 #27.

thanks

**Comment by flj151** [comment]
> @liuruijin17 How can I use in my owndata? I create a json file, and write in my data path, ![Dingtalk_20210413093733](https://user-images.githubusercontent.com/29905731/114485322-4c7b8180-9c3e-11eb-8cda-23fe13ba0f2e.jpg)
> 
> I use the command: "python test.py LSTR --testiter 500000 --modality eval --split testing --debug" and get the wrong result as follow: ![AEBSdata_1492626047222176976_0_3](https://user-images.githubusercontent.com/29905731/114485327-50a79f00-9c3e-11eb-852e-9c42446dd1df.jpg)

Hello ,can you tell me how you create json file like this?Looking forward to your reply！Thank you  very much！

**Comment by tigerdhl** [comment]
> > @liuruijin17 How can I use in my owndata? I create a json file, and write in my data path, ![Dingtalk_20210413093733](https://user-images.githubusercontent.com/29905731/114485322-4c7b8180-9c3e-11eb-8cda-23fe13ba0f2e.jpg)
> > I use the command: "python test.py LSTR --testiter 500000 --modality eval --split testing --debug" and get the wrong result as follow: ![AEBSdata_1492626047222176976_0_3](https://user-images.githubusercontent.com/29905731/114485327-50a79f00-9c3e-11eb-852e-9c42446dd1df.jpg)
> 
> Hello ,can you tell me how you create json file like this?Looking forward to your reply！Thank you very much！
I just test some images by created Artificially

**Comment by flj151** [comment]
> I just test some images by created Artificially

How you create the json file like tusimple?

**Comment by tigerdhl** [comment]
> > I just test some images by created Artificially
> 
> How you create the json file like tusimple?

Artificially. I add some data into the json file.
